lambda/fetchAndValidate.ts

```diff
@@ -0,0 +1,20 @@
+import { type Static, type TObject } from '@sinclair/typebox'
+import { validateWithTypeBox } from '@hello.nrfcloud.com/proto'
+
+export const fetchAndValidate = async <Schema extends TObject>(
+	schema: Schema,
+	url: string,
+	apiKey: string,
```

Comment:
style: use parameter bags if a method has arguments that have the same type to make it harder to mix them up.